.github/workflows/tests.yml

```diff
@@ -0,0 +1,26 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                php-version: [8.0, nightly]
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@2.25.2
```

Comment:
Let's use docker following the workflow [here](https://github.com/utopia-php/database/blob/main/.github/workflows/tests.yml) so we don't need to use third-party actions

Reply:
@abnegate Done